### ISSUE TITLE
Show filenames with errors in --check mode

### ISSUE THREAD

**Issue by arxell** [PyCQA/autoflake#56]
Now:

```
Running autoflake...
Unused imports/variables detected.%
```

How will be great:
```
Running autoflake...
src/foo/bar/file.py - Unused imports/variables detected.
```

**Comment by MarcoGorelli** [comment]
Seems this was fixed in #57

**Comment by ericstengard** [comment]
This should've been fixed already but #148 fixed this again. @fsouza  can we close this?
